docs/configuration/external-databases.md

```diff
@@ -0,0 +1,18 @@
+# Using external databases
+
+## Cloud controller database
+
+You can use an external database for the cloud controller by applying an overlay like
```

Comment:
terminology nit pick: this is not an overlay. below examples just sets data values.

Reply:
That is of course right. You would probably want to apply those values as an overlay. I will change it.

Reply:
Hello Ralf,

Thanks for your contribution. Have you tested this with an external database? does it work? 

IMO, this should be in  `sample-cf-install-values.yml` as an option (the yml should be commented and have instruction for user).

Reply:
I am not sure I agree to move this to `sample-cf-install-values.yml`, because a YAML file is not really readable documentation and this particular configuration is optional.

Reply:
> Hello Ralf,
> 
> Thanks for your contribution. Have you tested this with an external database? does it work?
> 
> IMO, this should be in `sample-cf-install-values.yml` as an option (the yml should be commented and have instruction for user).

hey @c0d1ngm0nk3y, our PM is interested in hearing if you've tested this already.

Reply:
@jamespollard8 We did of course test it before that PR and it worked as expected. But we faced some problems when updating our cf-for-k8s. Not sure if this was really related, but we decided to revert using the external ccdb for now.